x-pack/plugins/security_solution_serverless/public/get_started/welcome_panel/change_plan_link.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { SecurityPageName } from '@kbn/security-solution-plugin/common';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  useEuiTheme,
+  useEuiBackgroundColorCSS,
+  EuiLink,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import type { ProductTier } from '../../../common/product';
+import { ProductTierBadge } from './product_tier_badge';
+import { WELCOME_PANEL_PROJECT_CREATED_CHANGE_PLAN_TITLE } from './translations';
+import { Spacer } from './spacer';
+import { getCloudUrl } from '../../navigation/links/util';
+import { useKibana } from '../../common/services';
+
+const ChangePlanLinkComponent = ({ productTier }: { productTier: ProductTier | undefined }) => {
+  const { euiTheme } = useEuiTheme();
+  const { cloud } = useKibana().services;
+  const colorStyles = useEuiBackgroundColorCSS();
+  const cssStyles = [colorStyles.primary];
```

Comment:
Can't we do `${colorStyles.primary};` directly inside the `css`? instead of creating an array